server/portal/apps/intromessages/views.py

```diff
@@ -34,3 +46,24 @@ def put(self, request, *args):
                 new_db_message = IntroMessages.objects.create(user=request.user, component=component_name, unread=component_value)
                 new_db_message.save()
         return JsonResponse({'status': 'OK'})
+
+
+@method_decorator(login_required, name='dispatch')
+class CustomMessagesView(BaseApiView):
```

Comment:
Similarly as above, would there be a case where we would want to access admin messages without authentication?